includes/html/graphs/device/bits.inc.php

```diff
@@ -22,6 +22,9 @@
 
     if (is_array(\LibreNMS\Config::get('device_traffic_descr'))) {
         foreach (\LibreNMS\Config::get('device_traffic_descr') as $ifdescr) {
+            if (is_null($ifdesc) || !is_string($ifdesc) || $ifdesc == '') {
```

Comment:
Shouldn't `empty()` be enough?

Reply:
I thought about it ... what if the ifdesc is a literal '0' ... empty would return true, another false positive.